electrification.mss

```diff
@@ -221,7 +214,7 @@
        line-color: #97FF2F;
     }
 
-    [frequency!=null][frequency!=0][voltage>=15000][voltage<25000] {
+    [frequency=null][frequency!=0][voltage>=15000][voltage<25000] {
```

Comment:
This change looks suspicious.

Reply:
It looks highly suspicious and downright wrong.  There is a bug covered in #42 / PR #57 that I may tried to briefly address but something may have gone wrong with the merging

The original code on line 224 is `[frequency=null][frequency!=0][voltage>=25000] ` which does nothing.  The code was supposed to assign a color to AC lines >25000 such as the mine-to-power plant lines in the SW United States.  

So the code should be `[frequency!=null][frequency!=0][voltage>=25000]` or maybe even '[frequency>0][voltage>=25000]' and I tried to briefly address this in my PR.  Perhaps since I might have merged the proposed changes from PR #57 and then derived changes, something is messed up here.

Reply:
Ok, the current code in the repo for electrification.mss on lines 212 and 224 is incorrect.  My submission doesn't change these lines even though they are wrong.  Essentially any AC line between 15 kV and 25 kV (like 20 kV in Japan) or any AC line above 25 kV is not rendered with a color right now.  While I haven't changed the code here, I was indeed working from another branch and the change highlighting makes it appear I've substituted bad code for good code when it's more like I'm leaving bad code but had a sample of good code that I'm letting PR #57 work out.

I could do a simple PR that addresses Issue #42 but is not as extensive as PR #57. PR #57 fixes these issues but also tries to add separate colors for major lines that are visible at zoom level 2 but aren't included in the existing color scheme, namely Japanese electrification (20 kV/50 Hz and 20 kV/60 Hz) and Metro-North/Amtrak 12.5 kV/60 Hz in New York and Connecticut.  I could then rebase after that PR is accepted.